static/app/components/codeSnippet.tsx

```diff
@@ -14,7 +10,7 @@ import space from 'sentry/styles/space';
 
 interface CodeSnippetProps {
   children: string;
-  language: keyof typeof Prism.languages;
+  language: string;
```

Comment:
Unfortunately there doesn't seem to be any type declarations for available Prism languages, [language names are simply typed as `string`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/prismjs/components/index.d.ts)

Reply:
Yeah that's unfortuante.